mudslide/constants.py

```diff
@@ -3,3 +3,4 @@
 
 boltzmann = 3.166811563455557e-6
 eVtoHartree = 27.211386245988
+amu_to_au = 1822.88825
```

Comment:
Where is this coming from? It's not the value I'm back calculating from NIST.

Reply:
edited - I use this site: 
<img width="328" alt="Screen Shot 2022-04-10 at 2 26 16 PM" src="https://user-images.githubusercontent.com/46786940/162634108-b0521046-7fb0-4e4e-ab02-d46fd0103672.png">

and here are my back calculations from NIST values.

<img width="1030" alt="Screen Shot 2022-04-10 at 2 28 02 PM" src="https://user-images.githubusercontent.com/46786940/162634122-fd52beb2-dd80-4ff9-8f0b-2cb4af43b827.png">